Nos/Views/Moderation/ContentFlagView.swift

```diff
@@ -0,0 +1,25 @@
+import SwiftUI
+
+/// Displays pickers for selecting content flag option category, with additional stages shown 
+/// based on previous selections.
+struct ContentFlagView: View {
+    @Binding var selectedFlagOptionCategory: FlagOption?
+
+    var body: some View {
+        ScrollView {
+            VStack {
+                FlagOptionPicker(
+                    selectedOption: $selectedFlagOptionCategory,
+                    options: FlagOption.flagContentCategories,
+                    title: String(localized: .localizable.reportContent),
```

Comment:
It looks like this displays "Report Content" in English, but the [Figma design](https://www.figma.com/design/s0qf4VmyQygydP8MIQazZc/Nos?node-id=8435-4994&t=1lWsTMZ0hYv86QCe-4) shows "Flag Content". Do we want to update this now or is there a plan to do it in a future PR?

Reply:
I will create new strings for this, since we still have the old menu in the codebase